addons/dockable_container/dockable_container.gd

```diff
@@ -77,24 +93,25 @@ func _input(event: InputEvent) -> void:
 
 
 func add_child(node: Node, legible_unique_name: bool = false) -> void:
-	.add_child(node, legible_unique_name)
+	super.add_child(node, legible_unique_name)
 	_drag_n_drop_panel.raise()
 	_track_and_add_node(node)
 
 
 func add_child_below_node(node: Node, child_node: Node, legible_unique_name: bool = false) -> void:
-	.add_child_below_node(node, child_node, legible_unique_name)
+	#super.add_child_below_node(node, child_node, legible_unique_name)
+	super.add_sibling(child_node, legible_unique_name)
 	_drag_n_drop_panel.raise()
 	_track_and_add_node(child_node)
 
 
 func remove_child(node: Node) -> void:
-	.remove_child(node)
+	super.remove_child(node)
 	_untrack_node(node)
 
 
 func can_drop_data_fw(position: Vector2, data, from_control) -> bool:
-	return from_control == _drag_n_drop_panel and _can_handle_drag_data(data)
+	return from_control == _drag_n_drop_panel and data is Dictionary and data.get("type") == "tabc_element"
 
 
 func drop_data_fw(position: Vector2, data, from_control) -> void:
```

Comment:
In the same way, in 4.0, this method is expected to be called `_drop_data_fw` instead of `drop_data_fw`.
Reference: https://docs.godotengine.org/en/latest/classes/class_control.html?highlight=control#class-control-method-set-drag-forwarding